lib/order.rb

```diff
@@ -0,0 +1,56 @@
+
+require 'colorize'
+
+class Order
+ 
+  ITALIAN = {
+    "Focaccia" => 4, 'Burrata' => 8, 'Salad' => 5, 'Brusscetta' => 5,
+    'Salami' => 6, 'Lasagne' => 5, 'Pizza Margherita' => 10,'Arrabiata' => 10,
+    'Penne a la Vodka' => 12, 'Canelloni' => 15, 'Pumpkin Ravioli' => 12,
+    'Eggplant Parmigiana' => 13, 'Spaghetti Puttanesca' => 10,
+    'Penne Pomodoro' => 9, 'Tiramisu' => 6, 'Afogato' => 7
+  }.freeze
+
+  attr_reader :my_order
+
+  def initialize
+    @my_order = []
+  end
+
+  def view_menu
+    ITALIAN.each do |key, value|
+      puts "#{key},  £#{value}".colorize(:yellow).center(100)
```

Comment:
A nice idea of using string interpolation to make the menu easy to read & user friendly!